src/com/pentalog/pentastagiu/homework2/problem3/User.java

```diff
@@ -0,0 +1,27 @@
+package com.pentalog.pentastagiu.homework2.problem3;
+
+public class User {
+
+    private String name;
+    private String email;
+    private boolean isRegistered = true;
+    private Message message;
```

Comment:
desi ai adaugat aici message ca field in User, nu te-ai folosit in implementare de el: ai metoda postMessage, dar acolo lucrezi cu un message trimis ca parametru, tinand cont ca ai ales sa faci astfel implementarea de test cu 2 arrayuri, unul de useri si unul de mesaje, nu mai e nevoie sa adaugi message aici
daca vrei sa evoluezi implementarea si tinand cont de faptul ca am facut la webinar si colectii poti sa faci astfel: sa adaugi in User o lista de messages(fiecare mesaj poate avea de exemplu si o data, pe langa autor si descriere) iar apoi in codul de test adaugi useri, fiecare avand o lista de mesaje iar cand faci in board display messages faci asa:
- din lista de useri, ii consideri doar pe cei care sunt inregistrati
- aduni mesajele de la toti userii si le aranjezi in ordine crescatoare a datei
- le printezi la consola